src/test/java/com/ebivariation/contigalias/FTPBrowserTest.java

```diff
@@ -56,30 +57,42 @@ void changeDirectory() throws IOException {
     void changeDirectoryAndList() throws IOException {
         FTPBrowser ftpBrowser = new FTPBrowser();
         try {
-            ftpBrowser.connect();
-            ftpBrowser.changeToDirectory("genomes");
-            FTPFile[] ftpFiles = ftpBrowser.listDirectories();
+            ftpBrowser.connect(server, null);
+            ftpBrowser.navigateToDirectory("genomes");
+            FTPFile[] ftpFiles = ftpBrowser.listFiles();
             assertTrue(ftpFiles.length > 0);
         } finally {
             ftpBrowser.disconnect();
         }
     }
-    
+
     @Test
     void changeToNestedDirectoryAndFindAssemblyReport() throws IOException {
         FTPBrowser ftpBrowser = new FTPBrowser();
         try {
-            ftpBrowser.connect();
-            ftpBrowser.changeToDirectory("genomes/all/GCA/000/002/305/GCA_000002305.1_EquCab2.0/");
-            FTPFile[] ftpFiles = ftpBrowser.listDirectories();
+            ftpBrowser.connect(server, null);
+            ftpBrowser.navigateToDirectory("genomes/all/GCA/000/002/305/GCA_000002305.1_EquCab2.0/");
+            FTPFile[] ftpFiles = ftpBrowser.listFiles();
             assertTrue(ftpFiles.length > 0);
             String assemblyReport = "GCA_000002305.1_EquCab2.0_assembly_report.txt";
             boolean found = Stream.of(ftpFiles)
-                                  .anyMatch(f -> f.getName().contains(assemblyReport));
+                    .anyMatch(f -> f.getName().contains(assemblyReport));
```

Comment:
our style guide recommends aligning, so keep this as it was before.

Here is an xml you can import into intellij so that ctrl+alt+i reformats according to our style.
[EVA-style-settings.v3.xml.zip](https://github.com/EBIvariation/contig-alias/files/4633150/EVA-style-settings.v3.xml.zip)

Reply:
I tried to set this up and it completely reset my IDE